servers/register_server_types.cpp

```diff
@@ -247,6 +248,13 @@ void register_server_types() {
 	GDREGISTER_CLASS(RDShaderFile);
 	GDREGISTER_CLASS(RDPipelineSpecializationConstant);
 
+	GDREGISTER_CLASS(Texture2DRD);
```

Comment:
We don't have a place to register `renderer_rd` classes in the `renderer_rd` folder. If the renderer is excluded this will fail. I'm also wondering why we are including `rendering_device` and declaring its binding classes outside of `renderer_rd`. 

One thing we discussed on chat was that `renderer_rd` should really be a module but that is a big change that we don't want to do right now.

I'm guessing I should just put an `#ifdef` around this block?

Reply:
I don't think we have a compile time macro to remove the RD renderer. So I think this is fine for now